src/Features/CSharp/Portable/UseDefaultLiteral/CSharpUseDefaultLiteralCodeFixProvider.cs

```diff
@@ -51,6 +51,12 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             var originalNodes = diagnostics.SelectAsArray(
                 d => (DefaultExpressionSyntax)originalRoot.FindNode(d.Location.SourceSpan, getInnermostNodeForTie: true));
 
+            // This code fix will not make changes that affect the semantics of a statement or declaration. Therefore,
+            // we can skip the expensive verification step in cases where only one default expression appears within the
+            // group.
+            var nodesBySemanticBoundary = originalNodes.GroupBy(node => GetSemanticBoundary(node));
+            var nodesToVerify = nodesBySemanticBoundary.Where(group => group.Skip(1).Any()).SelectMany(group => group).ToSet();
```

Comment:
:memo: Originally I implemented this as skipping the first element of *all* groups in `nodesBySemanticBoundary` (as opposed to skipping the first element of groups with only one element), which means the `nodesToVerify` set would be a subset of what you see here. While I don't see any reason why this would fail, I felt it was a slightly more risky change.